README.md

```diff
@@ -52,8 +52,10 @@ Ensure that you're sitting adjacent to the stick on the network and that you hav
 # Frontier, with an assigned FRX523
 ./fs_xgspon_mod.py install GPON227000fe frontier FTRO12ab34cd --eqvid FRX523
 
-# KPN, keeping the serial as-is and basically just changing the ethernet uni slot to 1
-# this may also 'just work' for any other ISP that just requires slot 1 to be used instead of 10
+# KPN, just changing the ethernet uni slot from 10 to 1 and keeping the rest as-is
+# you can register the serial number of the module through the self-service tool of your provider
+./fs_xgspon_mod.py overrideslot GPON227000fe 1
+# you can also opt to fully install this mod, which also gives you e.g. SSH support (but strictly not neccesary)
```

Comment:
would rather see this just be the install command, but mention that KPN can skip installation and go use the overrideslot command instead